test/pos.test.ts

```diff
@@ -28,6 +28,11 @@ describe('matched-positions', async () => {
             20,
             "[border=\\"gray4\\"]",
           ],
+          [
+            21,
+            22,
+            "[border=\\"2\\"]",
```

Comment:
Should be escape  like this `"[border=\"\\32 \"]"`

Reply:
Here it must be "2" and not "\32", but I forget why, wait for me to sleep first, I will give you an answer tomorrow.

Reply:
ok,  i have already push the  changes , you can undo it , if you consider it's wrong

Reply:
There are some context about this issue
https://drafts.csswg.org/cssom/#escape-a-character-as-code-point

Reply:
<img width="260" alt="image" src="https://user-images.githubusercontent.com/24505451/198820685-c7a2a1a6-bede-401d-ae2e-b33ad2764800.png">
@sudongyuer vscode already does escape (I'm guessing), so if we do it again, nothing will show up